maths/average_mode.py

```diff
@@ -1,7 +1,7 @@
 import statistics
 
 
-def mode(input_list):  # Defining function "mode."
+def mode(input_list: list) -> int:  # Defining function "mode."

```

Comment:
The input list could contain any types of values, not just integers. The mode function works perfectly for a list of strings as well (or even a mixed list). We don't really care about the type of the data. We just care how many times the data occurs in the list.

So, the return value of this function should be set to `typing.Any`.